hooks/cache_location.py

```diff
@@ -25,10 +26,12 @@ class CacheLocation(HookBaseClass):
     For further details, see individual cache methods below.
     """
     
-    def path_cache(self, project_id, pipeline_configuration_id):
+    def path_cache_v2(self, project_id, entry_point, pipeline_configuration_id):
```

Comment:
Perhaps we could be a little more explicit with this name too? Something like `get_path_cache_file_path` or some variation? I think I'd prefer that to `_v2` with a quick note in the docstrings about the transition?